<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PIM</class>
 <widget class="QWidget" name="PIM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>view personal information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify personal information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Recharge</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="moneyEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify the address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the amount you want to top up:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify the tele:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="teleEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify personal information</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
